src/openvic-simulation/modifier/Modifier.cpp

```diff
@@ -49,8 +49,8 @@ bool ModifierValue::empty() const {
 	return values.empty();
 }
 
-fixed_point_t ModifierValue::get_effect(ModifierEffect const* effect, bool* successful) {
-	const effect_map_t::const_iterator it = values.find(effect);
+fixed_point_t ModifierValue::get_effect(ModifierEffect const& effect, bool* successful) const {
```

Comment:
Rename `successful` to `has_value` or `is_present` or `has_effect` to clarify it's a lookup instead of a try catch scenario.